src/app/components/views/homepage/homepage.component.ts

```diff
@@ -88,7 +88,7 @@ export class HomepageComponent implements OnInit, OnDestroy {
       this.jobService.call(uuid);
     }
     else {
-      this.jobService.call(uuid, this.jobCount);
+      this.jobService.call(`${uuid}/${this.jobCount}`);
```

Comment:
The call method should truthy check required values prior to making post/get request to backend.  There is no validation on the individual items here unless in the service file it's checking for this string.  I suggest passing in two args `(uuid: string, jobCount: number (or string))`